README.md

```diff
@@ -2,28 +2,35 @@
 
 ![Screenshot](./img/qgis-planet.webp)
 
-By Tim Sutton and QGIS Contributors.
+This project aggregates various RSS feeds related to the project, providing a centralized hub for updates and news.
+
+It has been mainly developed by Tim Sutton, Lova Andriarimalala and contributors.
 
 ## Project Status
 
 [![Deploy Hugo site to Pages](https://github.com/qgis/QGIS-Planet-Website/actions/workflows/github-pages.yml/badge.svg)](https://github.com/qgis/QGIS-Planet-Website/actions/workflows/github-pages.yml)
 ![Website Status](https://img.shields.io/website-up-down-green-red/http/qgis.github.io/QGIS-Planet-Website.svg)
 ![License](https://img.shields.io/github/license/qgis/QGIS-Planet-Website.svg)
 
-![](https://img.shields.io/github/issues/qgis/QGIS-Planet-Website.svg)
```

Comment:
I think'let's keep the shields unless there is a clear reason for removing them...